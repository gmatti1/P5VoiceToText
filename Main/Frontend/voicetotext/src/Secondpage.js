import React from 'react';
import './App.css';
import 'bootstrap/dist/css/bootstrap.min.css';

function Secondpage() {
  
    return (
      <div className="Secondpage" id="main"> 
	  <p>helloooo page 2</p>
	  </div>

    );
  }

export default Secondpage;
