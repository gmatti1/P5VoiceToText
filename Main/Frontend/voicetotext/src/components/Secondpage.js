import React from './../../node_modules/react';
import './../styles/App.css';
import './../../node_modules/bootstrap/dist/css/bootstrap.min.css';
import FileUpload from '../containers/FileUpload';
import Home from '../containers/Home';
import ConvertedAndCategorizedText from './../containers/ConvertedAndCategorizedText';

function Secondpage() {
  return (
    <div className='Secondpage' id='main'>
      <FileUpload />
      <ConvertedAndCategorizedText />
    </div>
  );
}

export default Secondpage;
