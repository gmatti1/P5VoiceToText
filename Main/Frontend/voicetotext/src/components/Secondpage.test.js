import React from 'react';
import {render, cleanup} from '@testing-library/react';
import Secondpage from './Secondpage';

 /**
 *
 * This is the test file for Secondpage component.
 *
 * @version 1.0
 * @author [Shashidhar Reddy Vanteru] <svanter1@asu.edu>
 * @copyright [Copyright 2020, P5VoiceToText]
 * 
 */

afterEach(cleanup)
it('should take a snapshot', () => {
	const { asFragment } = render(<Secondpage />)
	expect(asFragment(<Secondpage />)).toMatchSnapshot()
});

