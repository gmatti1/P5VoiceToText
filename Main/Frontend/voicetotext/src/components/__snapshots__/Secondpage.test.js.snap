// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Secondpage"
    id="main"
  >
    <div>
      <form>
        <h1
          class="Uploadheader"
        >
          Please upload the audio file
        </h1>
        <input
          class="Input"
          name="voiceFile"
          required=""
          type="file"
        />
        <button
          class="Buttonformat"
          type="submit"
        >
          Upload
        </button>
      </form>
      <div
        class="ORtext"
      >
        or
      </div>
      <a>
        <button
          class="Historybutton"
          type="submit"
        >
          Go To History
          <span
            class="tooltiptext"
          >
            Previously uploaded files
          </span>
        </button>
      </a>
      <div
        class="Textdata"
      >
        <label
          class="LabelTextdata"
        >
          Converted Text
        </label>
        <div />
        <label
          class="Confi"
        >
          Text Confidence : 0%
        </label>
      </div>
      <div
        class="Categorydata"
      >
        <label
          class="LabelTextdata"
        >
          Categorized Text
        </label>
      </div>
      <button
        class="btn btn-danger"
        id="calltopopup"
      >
        Add keyword for IMIST-AMBO
      </button>
    </div>
  </div>
</DocumentFragment>
`;
